lib/vsc/utils/py2vs3/py2.py

```diff
@@ -133,3 +133,28 @@ def _rmtree(self, path):
             self._rmdir(path)
         except OSError:
             pass
+
+
+def which(cmd, mode=os.F_OK | os.X_OK, path=None):
+    """
+    Return the first absolute path that matches the given command.
+
+    Searches via specified path, or $PATH as fallback.
```

Comment:
nvm my remarks if this is the py3 code

Reply:
This is the implementation for Python 2, but the function signature here is such that it matches `shutil.which` (supported since Python 3.3), and there the `path` value is expected to be a string value (like the raw `$PATH` value)